seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*[\\s]+style\\s*?=\\s*('|\\\")?\\s*([\\s\\S]*;+|)*\\s*color\\s*?:\\s*?white\\s*;*\\s*('|\\\")?/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
These tests are parsed from strings, so everything that would required escaping in strings, will need double escaped to make it through the parse. `('|\\\")` should be `(\\'|\\\")`

` ([\\s\\S]*;+|)*` I don't understand this? You are asking it to match everything(zero to unlimited times) before `;`(one to unlimited times) or nothing(zero to unlimited times). `;` is matched by `\S`, so why single it out? 

Your current regExp will match:

```
<h1 kljbvfo;ad


bvo;abv;lkkbnsv;;;;;;;;;;;; style = color: white ;
```